src/node_state/verifiers.rs

```diff
@@ -96,4 +128,33 @@ mod tests {
             Err(e) => panic!("Risc0 verification failed: {:?}", e),
         }
     }
+
+    #[test_log::test(test)]
+    fn test_sp1_proof_verifier() {
+        let encoded_proof = load_encoded_receipt_from_file("./tests/proofs/sp1_basic_proof.bin");
+        let verification_key = serde_json::to_string(&json!({
+            "vk":{
+                "commit":{
+                    "value":[575007420,1261203209,1098152299,719575249,1753499905,1551016589,1342492089,1942151841],"_marker":null},
+                    "pc_start":2105004,
+                    "chip_information":[["MemoryProgram",{"log_n":19,"shift":1},{"width":6,"height":524288}],["Program",{"log_n":19,"shift":1},{"width":37,"height":524288}],["Byte",{"log_n":16,"shift":1},{"width":11,"height":65536}]],
```

Comment:
how this was generated & the bin was generated ? We might need to re-generate those whenever we change the HyleOutput...

Reply:
hmm en effet, en l'état c'est un poil chiant de la régénerer. Je compte faire un bout de refacto derrière je vais voir si je change avant ou si je change après